docker/deb-rpm-s3/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+#
+# Build script for the deb-rpm-s3 docker container.
+#
+
+cd "$(dirname "$0")"
+
+if [ ! -e "elasticsearch.asc" ]; then
+    cat << EOF
+You must place a copy of the Elasticsearch GPG signing key (named
+elasticsearch.asc) into
```

Comment:
This means that the key will be in the Docker image? We should be careful then not to push it to a public Docker registry, right?